### DISCUSSION TITLE
Ability to highlight text

### DISCUSSION THREAD

**Discussion by igloows** [spacecowboy/Feeder#343] [Ideas]
It would be useful to have the ability to highlight text. And being able to export the highlighted excerpts with the original URL included.

**Comment by spacecowboy** [comment]
Selecting text is already possible. Just long press like in any app

**Reply by igloows** [reply]
I meant not just selecting but highlighting the text with a different color, usually yellow,  and to keep the highlighted text for future reference. Something like a yellow marker on a paper so that you can go back whenever you want to the article you read and see what you highlighted.

**Reply by spacecowboy** [reply]
ah I see
